src/DnsUtils.ts

```diff
@@ -0,0 +1,53 @@
+import DnsRecordsError, { DnsRecordsErrorCode } from "./errors/dnsRecordsError";
+import { CryptoRecords, DnsRecord, DnsRecordType } from "./publicTypes";
+import { isStringArray } from "./utils";
+
+export default class DnsUtils {
+
+  public toClassical(record: CryptoRecords): DnsRecord[] {
+    const dnsTypes = this.getAllDnsTypes(record);
+    return ([] as DnsRecord[]).concat(
+      ...dnsTypes.map(type => this.constructDnsRecords(record, type))
+    );
```

Comment:
Concatenation seems to be redundant

Reply:
This is because the map returns me a DnsRecord[][] and I was looking for some ways to flatten the array. As per a single type, there might be many records.

Reply:
I believe you can do just `[...dnsType.map(/* ... */)]`